routes/city.js

```diff
@@ -542,4 +537,36 @@ router.post("/spec-ds", tokenAuthentication, async (req, res) => {
   }
 });
 
+router.get("/jobs", async (req, res) => {
+  const input = req.query;
+  async function getJobs(params) {
+    try {
+      const response = await axios.get(
+        `https://indreed.herokuapp.com/api/jobs`, {
+          params
```

Comment:
Is that possible to filter it and get just tech jobs? Or maybe there are API parameters?
Our project has tech direction

Reply:
Yes it is. You can use q=Software for Software Jobs, or whatever you need